lstore/db.py

```diff
@@ -23,7 +137,10 @@ def create_table(self, name, num_columns, key_index):
         if self.tables.get(name) is not None:
             raise NameError(f"Error creating Table! Following table already exists: {name}")
         
-        self.tables[name] = Table(name, num_columns, key_index)
+        if self.path is None:
```

Comment:
this is an error when creating db, not when creating table.
What you could do is assign a default db_path if a path isn't provided.